test-clients/src/main/resource/contract/performance/performanceContracts.txt

```diff
@@ -0,0 +1,157 @@
+## contracts generated by CertVEM
+## CartEVM  --gas-limit 300000 --size-limit=5120 --bytecode --initcode --output-dir test-clients/src/main/resource/contract/performance
+add_small.bin
+addmod_small.bin
+address.bin
+and.bin
+balance.bin
+balance_gas.bin
+blockhash.bin
+byte.bin
+calldatacopy.bin
+calldataload.bin
+calldatasize.bin
+caller.bin
+callvalue.bin
+codecopy.bin
+codesize.bin
+coinbase.bin
+create.bin
+create2.bin
+difficulty.bin
+div_small.bin
+dup1.bin
+dup2.bin
+dup3.bin
+dup4.bin
+dup5.bin
+dup6.bin
+dup7.bin
+dup8.bin
+dup9.bin
+dup10.bin
+dup11.bin
+dup12.bin
+dup13.bin
+dup14.bin
+dup15.bin
+dup16.bin
+eq.bin
+exp_small.bin
+extcodecopy.bin
+extcodecopy_gas.bin
+extcodehash.bin
+extcodehash_gas.bin
+extcodesize.bin
+extcodesize_gas.bin
+gas.bin
+gaslimit.bin
+gasprice.bin
+gt.bin
+isZero_small.bin
+isZero_zero.bin
+jump.bin
+jumpdest.bin
+jumpi.bin
+jumpi2.bin
+keccak.bin
+log0.bin
+log1.bin
+log2.bin
+log3.bin
+log4.bin
+lt.bin
+mload.bin
+mod_small.bin
+msize.bin
+mstore.bin
+mstore8.bin
+mul_small.bin
+mulmod_small.bin
+not.bin
+number.bin
+or.bin
+origin.bin
+pc.bin
+pop.bin
+push1.bin
+push2.bin
+push3.bin
+push4.bin
+push5.bin
+push6.bin
+push7.bin
+push8.bin
+push9.bin
+push10.bin
+push11.bin
+push12.bin
+push13.bin
+push14.bin
+push15.bin
+push16.bin
+push17.bin
+push18.bin
+push19.bin
+push20.bin
+push21.bin
+push22.bin
+push23.bin
+push24.bin
+push25.bin
+push26.bin
+push27.bin
+push28.bin
+push29.bin
+push30.bin
+push31.bin
+push32.bin
+sar.bin
+sdiv_small.bin
+sgt.bin
+shl.bin
+shr.bin
+signextend_small.bin
+sload.bin
+sload_gas.bin
+slt.bin
+smod_small.bin
+sstore.bin
+sstore_gas.bin
+sub_small.bin
+swap1.bin
+swap2.bin
+swap3.bin
+swap4.bin
+swap5.bin
+swap6.bin
+swap7.bin
+swap8.bin
+swap9.bin
+swap10.bin
+swap11.bin
+swap12.bin
+swap13.bin
+swap14.bin
+swap15.bin
+swap16.bin
+timestamp.bin
+xor.bin
+
+## Contract dependant
```

Comment:
These are the contracts that will depend on other deployed contracts.  I'll need to do some search/replace to apply the hedera addresses.